packages/nc-gui-v2/components/cell/MultiSelect.vue

```diff
@@ -19,6 +20,8 @@ const editEnabled = inject(EditModeInj, ref(false))
 const active = inject(ActiveCellInj, ref(false))
 
 const selectedIds = ref<string[]>([])
+const aselect = ref<any>(null)
```

Comment:
Instead of using `any` type you could just use the `Select` component as a type.
```ts
import type { Select as AntSelect } from 'ant-design-vue'

const aSelect = ref<typeof AntSelect>()
```

Also the `null` initializer is unnecessary.